### DISCUSSION TITLE
how to compiler c classes to asm

### DISCUSSION THREAD

**Discussion by FxamarAboali** [openssl/openssl#23223] [Q&A]
how to compiler c language to asm languge for openssl build

**Comment by t8m** [comment]
I am sorry but the question does not make sense to me. There is no need to compile c classes to asm during openssl build.

**Reply by FxamarAboali** [reply]
I edit some classes c but when I build openssl l can't see my change for result in android ics_openvpn app .

How to build openssl no-asm in android build
